base_location_geonames_import/README.rst

```diff
@@ -1,87 +1,112 @@
+    :target: https://runbot.odoo-community.org/runbot/134/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds a wizard to import cities and/or city zip entries from
 `Geonames <http://www.geonames.org/>`_ database.
 
+**Table of contents**
+
+.. contents::
+   :local:
+
 Installation
 ============
 
-To install this module, you need the Python library 'requests'.
+To install this module, you need the Python library 'requests'::
+
+    pip install requests
+
 
 Configuration
 =============
 
-To access the menu to import better zip entries from Geonames,
-you must add yourself to the groups *Technical features* and *Sales manager*.
+To access the menu to import city zip entries from Geonames
+you must add yourself to the groups *Administration / Settings*.
 
-If want want/need to modify the default URL
+If you want/need to modify the default URL
 (http://download.geonames.org/export/zip/), you can set the *geonames.url*
 system parameter.
 
 Usage
 =====
 
-Go to *Settings > Technical > Cities/Locations Management > Import from Geonames*,
+Go to *Contacts > Configuration > Localization > Import from Geonames*,
 and click on it to open a wizard.
 
-When you start the wizard, it will ask you to select a country. If the
-country has been set-up to require the entry of cites from a list you will be
-able to indicate if you want to import the cities or zip codes.
-
+When you start the wizard, it will ask you to select a country.
 
-Then, for the
-selected country, it will delete all the current better zip entries, download
+Then, for the selected country, it will delete all the current city zip entries, download
 the latest version of the list of cities from geonames.org and create new
-better zip entries.
-
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/134/11.0
+city zip entries.
 
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues
-<https://github.com/OCA/partner-contact/issues>`_. In case of trouble, please
-check there if your issue has already been reported. If you spotted it first,
-help us smashing it by providing a detailed and welcomed feedback.
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/partner-contact/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/partner-contact/issues/new?body=module:%20base_location_geonames_import%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
 
 Credits
 =======
 
+Authors
+~~~~~~~
+
+* Akretion
+* Agile Business Group
+* Tecnativa
```

Comment:
You deserve co-authorship in this module.

Reply:
Oh, I see you haven't updated README, but included in the manifest. Just update it :wink: